test/core/client_channel/lb_policies_test.c

```diff
@@ -780,15 +782,18 @@ static void verify_total_carnage_round_robin(const servers_fixture *f,
     }
   }
 
-  /* no server is ever available. The persistent state is TRANSIENT_FAILURE */
+  /* no server is ever available. The persistent state is TRANSIENT_FAILURE. May
+   * also be CONNECTING if, under load, this check took too long to run and some
+   * subchannel already transitioned to retrying. */
   for (size_t i = 0; i < sequences->n; i++) {
     const grpc_connectivity_state actual = sequences->connectivity_states[i];
-    const grpc_connectivity_state expected = GRPC_CHANNEL_TRANSIENT_FAILURE;
-    if (actual != expected) {
+    const uint32_t expected_bitset =
+        GRPC_CHANNEL_TRANSIENT_FAILURE | GRPC_CHANNEL_CONNECTING;
```

Comment:
The enum values are not bits, they're sequential values (i.e., the values are not 1, 2, 4, 8, etc; they are 0, 1, 2, 3, etc).  I don't think you can actually 'or' them together this way.  I think you actually need to check whether actual is equal to one or the other.